public/index.html

```diff
@@ -0,0 +1,156 @@
+          </button>
+        </span>
+      </div>
+      <div id="collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordian">
+        <div class="card-body" id="getSeries">
+          <div class="title">Method</div>
+          <div class="get answer">GET</div>
+          <div class="title">Route</div>
+          <div class="answer">http://localhost:4004/api/whedonverse/series</div>
+          <div class="title">Description</div>
+          <div class="answer">Returns a list of all of Joss Whedon's televison series. </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="card">
+      <div class="card-header" id="headingFour">
+        <span class="command">
+          <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseFour"
+            aria-expanded="false" aria-controls="collapseFour">
+            <span class="get">GET</span>/whedonverse/{actor}
+          </button>
+        </span>
+      </div>
+      <div id="collapseFour" class="collapse" aria-labelledby="headingFour" data-parent="#accordian">
+        <div class="card-body" id="getactor">
+          <div class="title">Method</div>
+          <div class="get answer">GET</div>
+          <div class="title">Route</div>
+          <div class="answer">http://localhost:4004/api/whedonverse/actor</div>
+          <div class="title">Description</div>
+          <div class="answer">Returns a list of all of the projects of Joss Whedon's that the specified actor has
+            starred in. </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="card">
+      <div class="card-header" id="headingFive">
+        <span class="command">
+          <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseFive"
+            aria-expanded="false" aria-controls="collapseFive">
+            <span class="post">POST</span>/whedonverse
+          </button>
+        </span>
+      </div>
+      <div id="collapseFive" class="collapse" aria-labelledby="headingFive" data-parent="#accordian">
+        <div class="card-body" id="postWhedonverse">
+          <div class="title">Method</div>
+          <div class="post answer">POST</div>
+          <div class="title">Route</div>
+          <div class="answer">http://localhost:4004/api/whedonverse</div>
+          <div class="title">Headers</div>
+          <div class="answer">Content-Type: application/json</div>
+          <div class="title">Body</div>
+          <div class="answer">{ "series": "New Series", "starring": ["Actors"], "airdates": "start year - end year",
+            "seasons": 1
+            }</div>
+          <div class="answer">{ "movie": "New Movie", "starring": ["Actors"], "releaseDate": "year", "director":
+            "Director" }</div>
+          <div class="title">Description</div>
+          <div class="answer">Adds a new entry into the WhedonVerse. </div>
+        </div>
+      </div>
+    </div>
```

Comment:
I would make this two different routes. It will be a very complicated controller otherwise.

POST http://localhost:4004/api/whedonverse/series

POST http://localhost:4004/api/whedonverse/movie